web/src/modules/theme/constants/dimensions.ts

```diff
@@ -18,6 +15,5 @@ const dimensions: DimensionsType = {
   toolbarWidth: 125,
   headerHeightLarge: 90,
   headerHeightSmall: 70
-}
-
-export default dimensions
+};
+export default dimensions;
```

Comment:
Did you run prettier?
```suggestion
export default dimensions
```